active/tab/tabcontrol.md

```diff
@@ -0,0 +1,580 @@
+
+The TabView control is a way to display a set of tabs and their respective content. Tab controls are useful for displaying several pages (or documents) of content while giving a user the capability to rearrange, open, or close new tabs. 
+
+![Tabs in Edge](https://user-images.githubusercontent.com/25991996/52679758-f6d6ad80-2eea-11e9-9955-fd111c26d982.png)
+
+Tab-like UI comes in two distinct styles that affect not only the visualization of the control, but also the user experience:
+* Static tabs
+    * Generally display a set number of pages
+    * User can change between pages, but cannot open new pages, close pages, or rearrange pages
+* Document tabs
+    * Display a variable number of pages
+    * User can add, remove, and rearrange pages
+    * User can "tear off" tabs into their own window or "recombine" tabs from one group into another group
+
+The Xaml platform already provides several controls to represent "static-style tabs" including [Pivot](https://docs.microsoft.com/en-us/windows/uwp/design/controls-and-patterns/pivot) and top [NavigationView](https://docs.microsoft.com/en-us/windows/uwp/design/controls-and-patterns/navigationview#display-modes). However, these controls do not well support "document-style" tabs due to a variety of limitations, including:
+* Significant effort up to (and including) retemplating to build simple tabs
+* No built-in support for closing tabs
+* No built-in support for drag/drop into new windows
+* No built-in support for moving a tab from a window and combining it with another window
+* Limited keyboard and accessibility support
+
+This spec therefore covers "document-style tabs" specifically as this platform gap isn't easily solved with an app-side workaround. 
+
+# Description
+<!-- Use this section to provide a brief description of the feature.
+For an example, see the introduction to the PasswordBox control 
+(http://docs.microsoft.com/windows/uwp/design/controls-and-patterns/password-box). -->
+
+TabView is a collection of tabs that each represents a new page or document in your app. Use TabView when your app has several pages of content and the user expects to be able to add, close, and rearrange the pages.
+
+**Is this the right control?**
+
+Use a TabView to help the user manage multiple app pages or documents within the same window. 
+
+Do not use a TabView to display a static set of tabs that the user cannot rearrange, open, or close. Use a Pivot or top NavigationView instead. 
+
+# Examples
+<!-- Use this section to explain the features of the API, showing
+example code with each description. The general format is: 
+  feature explanation,
+  example code
+  feature explanation,
+  example code
+  etc.-->
+
+<!-- As an example of this section, see the Examples section for the PasswordBox control 
+(https://docs.microsoft.com/windows/uwp/design/controls-and-patterns/password-box#examples). -->
+
+## To replicate the behavior of Microsoft Edge
```

Comment:
Don't oversell this. It's not a perfect replica (doesn't support needy state, for example). Perhaps

> To get behavior typical of a Web browser